releases/mainnet/1.2.4/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.2.4</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.4</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.4/contracts.json">Contract addresses</a></li>
+				<!-- <li><a href="https://ipfs.io/ipfs/QmeWh3Bd2m5uihhUidFR22PgLKzwYD5i54zUQYVeVVnLgK">Contract addresses (IPFS hosted)</a></li> -->
+				<!-- <li><a href="/releases/mainnet/1.2.4/abi/index.html">ABIs</a></li> -->
+				<li>Links to GitHub repositories</li>
+					<ul>
+						DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.4" target="_blank">changelog</a>)</li>
```

Comment:
DSS scripts release not available. Either comment it out or wait for it before merging.